assets/files/usr/local/bin/fletcher8

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
```

Comment:
+#!/usr/libexec/platform-python
^^ needs to be changes as of the change to ootpa

Reply:
I'm moving this to an init container in #296, there it will be okay with /usr/bin/env/python. I'll rebase it as soon as #296 is merged.

Reply:
Done. Thanks!